ops/services/app_functions/report_stream_batched_publisher/infra/main.tf

```diff
@@ -68,25 +68,26 @@ resource "azurerm_function_app" "functions" {
-    AZ_STORAGE_ACCOUNT_KEY         = data.azurerm_storage_account.app.primary_access_key
-    AZ_STORAGE_QUEUE_CXN_STRING    = data.azurerm_storage_account.app.primary_connection_string
-    TEST_EVENT_QUEUE_NAME          = var.test_event_queue_name
-    REPORTING_EXCEPTION_QUEUE_NAME = var.reporting_exception_queue_name
-    REPORT_STREAM_URL              = local.report_stream_url
-    REPORT_STREAM_TOKEN            = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.datahub_api_key.id})"
-    REPORT_STREAM_BATCH_MINIMUM    = var.report_stream_batch_minimum
-    REPORT_STREAM_BATCH_MAXIMUM    = var.report_stream_batch_maximum
-    SIMPLE_REPORT_CB_URL           = local.simple_report_callback_url
-    SIMPLE_REPORT_CB_TOKEN         = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.simple_report_callback_token.id})"
+    https_only                                           = true
+    FUNCTIONS_WORKER_RUNTIME                             = "node"
+    WEBSITE_NODE_DEFAULT_VERSION                         = "~14"
+    FUNCTION_APP_EDIT_MODE                               = "readonly"
+    HASH                                                 = "${base64encode(filesha256("${local.function_app_source}"))}"
+    WEBSITE_RUN_FROM_PACKAGE                             = "https://${data.azurerm_storage_account.app.name}.blob.core.windows.net/${azurerm_storage_container.deployments.name}/${azurerm_storage_blob.appcode.name}${data.azurerm_storage_account_sas.sas.sas}"
+    APPINSIGHTS_INSTRUMENTATIONKEY                       = data.azurerm_application_insights.app.instrumentation_key
+    AZ_STORAGE_QUEUE_SVC_URL                             = "https://${data.azurerm_storage_account.app.name}.queue.core.windows.net/"
+    AZ_STORAGE_ACCOUNT_NAME                              = data.azurerm_storage_account.app.name
+    AZ_STORAGE_ACCOUNT_KEY                               = data.azurerm_storage_account.app.primary_access_key
+    AZ_STORAGE_QUEUE_CXN_STRING                          = data.azurerm_storage_account.app.primary_connection_string
+    TEST_EVENT_QUEUE_NAME                                = var.test_event_queue_name
+    REPORTING_EXCEPTION_QUEUE_NAME                       = var.reporting_exception_queue_name
+    REPORT_STREAM_URL                                    = local.report_stream_url
+    REPORT_STREAM_TOKEN                                  = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.datahub_api_key.id})"
+    REPORT_STREAM_BATCH_MINIMUM                          = var.report_stream_batch_minimum
+    REPORT_STREAM_BATCH_MAXIMUM                          = var.report_stream_batch_maximum
+    SIMPLE_REPORT_CB_URL                                 = local.simple_report_callback_url
+    SIMPLE_REPORT_CB_TOKEN                               = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.simple_report_callback_token.id})"
+    "AzureWebJobs.ReportStreamExceptionHandler.Disabled" = "1"
```

Comment:
Only added this line, the others are a format change

Reply:
I think technically you don't have to quote this but I think it works either way. Did you get a chance to deploy this to a test env? I'm curious how the env var shows up like this.

Reply:
I actually do, I tried it without quotes and TF complained about not being able to fine `AzureWebJobs` resource

Reply:
Before on test
![Screenshot from 2022-02-17 15-48-00](https://user-images.githubusercontent.com/4952042/154577355-b1f772d4-c20e-4c1d-ae9b-25122f43d558.png)

After on test
![image](https://user-images.githubusercontent.com/4952042/154577330-94232140-1206-4c8a-874b-e06ef6595435.png)